docs/documentation.md

```diff
@@ -1,56 +1,65 @@
 # Writing Documentation
 
-[CiviCRM.org/documentation](https://civicrm.org/documentation) has a nice high-level list of all active documentation.
+[CiviCRM.org/documentation](https://civicrm.org/documentation) has a nice
+high-level list of all active documentation.
+
+## Wiki migration
+
+As of early 2017, developer documentation on the [wiki] is under
+active migration to mkdocs. If you are helping to migrate wiki pages, please
+read about the [migration process][migration] to ensure that you understand the
+workflow including the process of redirecting wiki pages to mkdocs.
+
+[migration]: https://wiki.civicrm.org/confluence/display/CRMDOC/Content+migration+from+wiki+to+Developer+Guide
+[wiki]: https://wiki.civicrm.org/confluence/display/CRMDOC/CiviCRM+Documentation
 
 ## Guides in mkdocs
 
-We are using [mkdocs](http://www.mkdocs.org) to produce guides, and currently have the following two:
+We are using [mkdocs](http://www.mkdocs.org) to produce guides, and currently
+have the following two:
 
-* [User Guide](https://docs.civicrm.org/user/en/stable/)
-* [Developer Guide](https://docs.civicrm.org/dev/en/master/) *(which you are reading now!)*
+-   [User Guide](https://docs.civicrm.org/user/en/stable/)
+-   [Developer Guide](https://docs.civicrm.org/dev/en/master/) *(which you are
+    reading now!)*
 
-The content for each of these guides is written in markdown, stored in text files, and hosted on GitHub.
+The content for each of these guides is written in [markdown], stored in text
+files, and hosted on GitHub.
 
 ### How to edit
 
-For minor changes you can simply edit the markdown online using GitHub. However, for a better editing experience we highly recommend installing `mkdocs`:
+For minor changes you can simply edit the [markdown] online using GitHub.
+However, for a better editing experience we highly recommend installing
+`mkdocs` as follows.
 
 1.  Obtain the source files for the guide you want to edit
-    1. Find the repository on GitHub (e.g. here is [the repo for this guide](https://github.com/civicrm/civicrm-dev-docs))
-    1. Fork and clone locally. (For more help with Git and GitHub see [Git](git))
+    1.  Find the repository on GitHub (see bottom left of screen)
```

Comment:
@totten my original motivation was to generalize the instructions to apply equally to the Dev Guide and the User Guide. (When I wrote these instructions originally, I hadn't yet figured out how to get the repo link to show up in the Dev Guide at the bottom left of the screen like it does in the User Guide.) But, now that you mention it, this change done indeed have the added benefit of requiring one fewer edit if/when the repo moves locations.